packages/compiler/src/server/serverlib.ts

```diff
@@ -1089,6 +1090,23 @@ export function createServer(host: ServerHost): Server {
           };
           host.log(sLog);
         },
+        trackAction: async (asyncAction: () => Promise<any>, startMessage: ProcessedLog) => {
+          host.log({
+            level: startMessage.level,
+            message: formatLog(startMessage, { excludeLogLevel: true }),
+          });
+
+          try {
+            return await asyncAction();
+          } finally {
+            getChildLogs().forEach((log) =>
+              host.log({
+                level: "info",
```

Comment:
any reason not to use startMessage.level? If the startMessage.level is lower than "info", user will see child logs without the startMessage which may be confusing.

Reply:
Hi Rodge, removed this update on serverlib.ts to avoid overlogging